grid:
  num_maps: 4
  min_surrounding_obstacles: 2
  cell_size_millis: 50
  step_tolerance: 0.5
  max_view_dist: 2700
  min_view_dist: 470

flags:
  onOdroid: False
  useOpenGL: True
  output: True

frame_id: "/kinect_link"

topic: "/IRIS/obstacles"
